registry/bin/boot

```diff
@@ -35,8 +37,7 @@ function etcd_set_default {
   ERROR="$(etcdctl --no-sync -C "$ETCD" mk "$ETCD_PATH/$1" "$2" 2>&1 >/dev/null)"
 
   if [[ $? -ne 0 ]] && echo "$ERROR" | grep -iqve "key already exists"; then
-    echo "etcd_set_default: an etcd error occurred ($ERROR)"
-    echo "aborting..."
+    echo "registry: etcd_set_default($1/$2) aborting, an etcd error occurred ($ERROR)"
```

Comment:
I would suggest `etcd_set_default($1=$2)` so we get:
```
etcd_set_default(foo=bar) aborting, an etcd error occurred...
```
Either that or a colon-plus-space `($1: $2)`.